cmd/root/auth.go

```diff
@@ -77,20 +77,6 @@ func MustAccountClient(cmd *cobra.Command, args []string) error {
 		cfg.Profile = profile
 	}
 
-	if cfg.Profile == "" {
-		// account-level CLI was not really done before, so here are the assumptions:
-		// 1. only admins will have account configured
-		// 2. 99% of admins will have access to just one account
-		// hence, we don't need to create a special "DEFAULT_ACCOUNT" profile yet
-		_, profiles, err := databrickscfg.LoadProfiles(cmd.Context(), databrickscfg.MatchAccountProfiles)
```

Comment:
To retain behavior in the described case (many admins will have only a single account defined), we could check `os.IsNotExist` and fall through if that's the error. WDYT?

Reply:
Agreed, added in a second revision